api-report/tree.api.md

```diff
@@ -835,8 +919,11 @@ export class SimpleDependee implements Dependee {
 // @public (undocumented)
 export function singleTextCursor(root: JsonableTree): TextCursor;
 
+// @public (undocumented)
+export type Skip = number;
+
 // @public
-type Skip = number;
+type Skip_2 = number;
```

Comment:
I'm not sure why API extractor did this (this has happened before; I think it's a bug?). Not sure if we want to try to work around it or if we don't care now (or ever). Up to you, rest of the PR looks good to me

Reply:
It does then when there are multiple types with the same name. Generally, this is not an issue though it does suggest you might want to tweak the naming.

Reply:
Ah I see, this exposed a second Skip.